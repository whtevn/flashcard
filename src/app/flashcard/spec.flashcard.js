import renderer from 'react-test-renderer';
import React from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import FCD_Flashcard from './component.flashcard.js';

test('Full flash card renders correctly', () => {
  const tree = renderer.create(
      <MuiThemeProvider>
        <FCD_Flashcard answer="answer" question="question" onCorrect={()=>{}} onIncorrect={()=>{}}  />
      </MuiThemeProvider>
      ).toJSON();
  expect(tree).toMatchSnapshot();
});

