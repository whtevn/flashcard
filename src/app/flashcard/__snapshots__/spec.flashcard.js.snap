// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Full flash card renders correctly 1`] = `
<div>
  <div>
    <div>
      question
    </div>
  </div>
  <div>
    <div>
      answer
      <div>
        <button
          className="MuiButtonBase-buttonBase-3170508663 MuiButton-root-3593367901 MuiButton-raised-1110834863 MuiButton-raisedContrast-2539079880"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          tabIndex="0"
          type="button"
        >
          <span
            className="MuiButton-label-49836587"
          >
            Incorrect
          </span>
          <span
            className="MuiTouchRipple-root-3868442396"
          />
        </button>
        <button
          className="MuiButtonBase-buttonBase-3170508663 MuiButton-root-3593367901 MuiButton-raised-1110834863"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          tabIndex="0"
          type="button"
        >
          <span
            className="MuiButton-label-49836587"
          >
            Correct
          </span>
          <span
            className="MuiTouchRipple-root-3868442396"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
